config/sync/core.entity_form_display.paragraph.image_grid.default.yml

```diff
@@ -18,7 +18,7 @@ content:
     settings:
       title: Paragraph
       title_plural: Paragraphs
-      edit_mode: closed
+      edit_mode: closed_expand_nested
```

Comment:
This also did the opposite of what needed to happen, the entire paragraph should be closed by default